Packs/cyberark_AIM/Integrations/CyberArkAIM_v2/CyberArkAIM_v2.py

```diff
@@ -151,13 +152,13 @@ def main():
         command = demisto.command()
         demisto.debug(f'Command being called in CyberArk AIM is: {command}')
 
-        commands = {
-            'test-module': test_module,
-            'cyberark-aim-list-credentials': list_credentials_command,
-            'fetch-credentials': fetch_credentials
-        }
-        if command in commands:
-            return_results(commands[command](client, demisto.args()))  # type: ignore[operator]
+        if command == 'test-module':
+            return_results(test_module(client))
+        elif command == 'cyberark-aim-list-credentials':
+            return_results(list_credentials_command(client))
+        elif command == 'fetch-credentials':
+            return_results(fetch_credentials(client, demisto.args()))
```

Comment:
or move the `demisto.credentials` here
```suggestion
           fetch_credentials(client, demisto.args())
```